index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <section class="toolbar">
+            <a id="logo" href="#">My Logo</a>
+            <div class="nav">
+                <img src="./assets/images/menu.png">
+            </div>
```

Comment:
- [ ] Nice work wrapping the header in a `header` tag :clap: However it would be nice to wrap the hamburger icon with `nav` tag for semantics reasons. and notice that our header is the "Navbar", so it would be good to only wrap the logo and the hamburger in with a `header` tag instead of wrapping everything in the header tag